tests/test_schema_validator.py

```diff
@@ -21,13 +21,13 @@
 }
 
 
-class TestJobType(StrEnum):
+class FakeJobType(StrEnum):
```

Comment:
This renaming is an unrelated, but necessary change. It turns out that pytest was failing at discovery, because it was interpreting this class as a test class, since it was starting with the keyword `Test` 🤦